app/src/atoms/MenuList/DropdownMenu.tsx

```diff
@@ -213,9 +216,12 @@ export function DropdownMenu(props: DropdownMenuProps): JSX.Element {
             width={width}
             top={dropdownPosition === 'bottom' ? '2.5rem' : undefined}
             bottom={dropdownPosition === 'top' ? '2.5rem' : undefined}
+            overflowY={OVERFLOW_AUTO}
+            maxHeight="20rem" // Set the maximum display number to 10.
           >
             {filterOptions.map((option, index) => (
               <MenuItem
+                zIndex="3"
```

Comment:
does this work? i thought zindexes are numbers

```suggestion
                zIndex={3}
```

Reply:
actually, it worked but yeah, I will switch to `{3}`.